src/test/index.test.ts

```diff
@@ -21,26 +26,97 @@ describe("Voting", () => {
     let sandboxInstance;
     let sponsoredPaymentMethod: SponsoredFeePaymentMethod;
 
+    let randomAccountManagers: AccountManager[] = [];
+    let randomWallets: AccountWallet[] = [];
+    let randomAddresses: AztecAddress[] = [];
+
+    let l1PortalManager: L1FeeJuicePortalManager;
+    let fundedAddressClaims: L2AmountClaim[] = [];
+    let feeJuiceAddress: AztecAddress;
+
     beforeAll(async () => {
         sandboxInstance = spawn("aztec", ["start", "--sandbox"], {
             detached: true,
             stdio: 'ignore'
         })
-        sleep(15000)
+        await sleep(15000);
         logger = createLogger('aztec:aztec-starter');
         logger.info("Aztec-Starter tests running.")
 
         pxe = await setupSandbox();
 
         wallets = await getInitialTestAccountsWallets(pxe);
-        accounts = wallets.map(w => w.getCompleteAddress())
+        accounts = wallets.map(w => w.getCompleteAddress());
         sponsoredPaymentMethod = await SponsoredFeePaymentMethod.new(pxe);
+
+        // generate random accounts
+        randomAccountManagers = await Promise.all(
+            (await generateSchnorrAccounts(5)).map(
+                a => getSchnorrAccount(pxe, a.secret, a.signingKey, a.salt)
+            )
+        );
+        // get corresponding wallets
+        randomWallets = await Promise.all(randomAccountManagers.map(am => am.getWallet()));
+        // get corresponding addresses
+        randomAddresses = await Promise.all(randomWallets.map(async w => (await w.getCompleteAddress()).address));
+
+        // create default ethereum clients
+        const nodeInfo = await pxe.getNodeInfo();
+        const chain = createEthereumChain(['http://localhost:8545'], nodeInfo.l1ChainId);
+        const DefaultMnemonic = 'test test test test test test test test test test test junk';
+        const { publicClient, walletClient } = createL1Clients(chain.rpcUrls, DefaultMnemonic, chain.chainInfo);
+
+        feeJuiceAddress = nodeInfo.protocolContractAddresses.feeJuice;
+
+        // create portal manager
+        l1PortalManager = await L1FeeJuicePortalManager.new(
+            pxe,
+            publicClient,
+            walletClient,
+            logger
+        );
+
     })
 
     afterAll(async () => {
         sandboxInstance!.kill();
     })
 
+    it("Creates accounts with fee juice", async () => {
+        // bridge funds to unfunded random addresses
+        const claimAmount = 10n ** 22n;
+
+        let claims: L2AmountClaim[] = [];
+        // bridge sequentially to avoid l1 txs (nonces) being processed out of order
+        for (let i = 0; i < randomAddresses.length; i++) {
+            claims.push(await l1PortalManager.bridgeTokensPublic(randomAddresses[i], claimAmount, true));
+        }
+        // arbitrary transactions to progress 2 blocks, and have fee juice on Aztec ready to claim
+        await EasyPrivateVotingContract.deploy(wallets[0], accounts[0]).send().deployed(); // deploy contract with first funded wallet
```

Comment:
I am not 100% sure if this is still needed. It'd be worth running the test with these commented out and see if it still passes.

Reply:
update: james checked and 2 txs are required